src/bitstring/fixed64.rs

```diff
@@ -13,25 +13,286 @@ A 64bit decimal number.
+        146, 110, 113, 78, 126, 168, 252, 37,
+    ]));
+
+    /// sqrt(2)
+    pub const SQRT_2: Self = Bitstring64(FixedBinaryBuf::from_le_bytes([
+        91, 204, 39, 238, 68, 20, 254, 37,
+    ]));
+
+    /// 1/sqrt(2)
+    pub const FRAC_1_SQRT_2: Self = Bitstring64(FixedBinaryBuf::from_le_bytes([
+        199, 170, 179, 184, 33, 135, 255, 33,
+    ]));
+
+    /// Euler's number (e)
+    pub const E: Self = Bitstring64(FixedBinaryBuf::from_le_bytes([
+        69, 100, 233, 210, 66, 152, 255, 41,
+    ]));
+
+    /// log<sub>2</sub>(10)
+    pub const LOG2_10: Self = Bitstring64(FixedBinaryBuf::from_le_bytes([
+        226, 61, 172, 133, 107, 161, 253, 45,
+    ]));
+
+    /// log<sub>2</sub>(e)
+    pub const LOG2_E: Self = Bitstring64(FixedBinaryBuf::from_le_bytes([
+        237, 185, 1, 196, 214, 66, 254, 37,
+    ]));
+
+    /// log<sub>10</sub>(2)
+    pub const LOG10_2: Self = Bitstring64(FixedBinaryBuf::from_le_bytes([
+        143, 140, 253, 105, 10, 129, 253, 33,
+    ]));
+
+    /// log<sub>10</sub>(e)
+    pub const LOG10_E: Self = Bitstring64(FixedBinaryBuf::from_le_bytes([
+        81, 53, 182, 160, 86, 52, 254, 33,
+    ]));
+
+    /// ln(2)
+    pub const LN_2: Self = Bitstring64(FixedBinaryBuf::from_le_bytes([
+        205, 102, 171, 200, 49, 59, 255, 33,
+    ]));
+
+    /// ln(10)
+    pub const LN_10: Self = Bitstring64(FixedBinaryBuf::from_le_bytes([
+        69, 120, 170, 195, 178, 130, 253, 41,
+    ]));
+}
+
+impl Bitstring64 {
+    /// The radix or base of the internal representation.
+    pub const RADIX: u32 = 10;
+
     /**
-    Create a decimal from the given buffer.
+    The number of digits in base 10 that can be represented without loss of precision.
 
-    The buffer is assumed to be in little-endian byte-order already.
+    This constant indicates the total count of significant decimal digits in the
+    significand, regardless of the decimal point's position. For instance
+    1234567 and 123.4567, both contain `DIGITS` digits.
     */
-    pub fn from_le_bytes(bytes: [u8; 8]) -> Self {
-        Self(FixedBinaryBuf::from(bytes))
+    pub const DIGITS: u32 = 16;
+
+    /**
+    [Machine epsilon] value.
+
+    This is the difference between `1.0` and the next larger representable number.
+
+    [Machine epsilon]: https://en.wikipedia.org/wiki/Machine_epsilon
+    */
+    pub const EPSILON: Self = Bitstring64(FixedBinaryBuf::from_le_bytes([1, 0, 0, 0, 0, 0, 0, 34]));
+
+    /// Smallest finite value.
+    pub const MIN: Self = Bitstring64(FixedBinaryBuf::from_le_bytes([
+        255, 252, 243, 207, 63, 255, 248, 247,
+    ]));
+
+    /// Smallest positive normal value.
+    pub const MIN_POSITIVE: Self =
+        Bitstring64(FixedBinaryBuf::from_le_bytes([1, 0, 0, 0, 0, 0, 0, 0]));
+
+    /// Largest finite value.
+    pub const MAX: Self = Bitstring64(FixedBinaryBuf::from_le_bytes([
+        255, 252, 243, 207, 63, 255, 248, 119,
+    ]));
+
+    /// Not a Number (NaN).
```

Comment:
Maybe we could call this out as _Not a Number, with a zero payload_? One of the goals of this project is to take advantage of the fact that NaN payloads are properly specified for decimal numbers, so we can safely transport them.

Reply:
Sure, that'd be better.